service-runner/src/main/resources/profiles/deployment/config.env

```diff
@@ -0,0 +1,62 @@
+# secrets
+secret.data.username=postgres
```

Comment:
The configuration values are stacked on top of the [default/config.env](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/service-runner/src/main/resources/profiles/default/config.env). 

That means, only values not defined in the `default/config.env` are needed.

Reply:
Fixed!